extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaTuningConfig.java

```diff
@@ -60,7 +60,9 @@ public KafkaTuningConfig(
   )
   {
     // Cannot be a static because default basePersistDirectory is unique per-instance
-    final RealtimeTuningConfig defaults = RealtimeTuningConfig.makeDefaultTuningConfig(basePersistDirectory);
+    final RealtimeTuningConfig defaults = new RealtimeTuningConfig.Builder()
```

Comment:
This isn't quite right, but flows from a mistake in the original code. KafkaTuningConfig shouldn't have a `basePersistDirectory`. It's not documented and it doesn't make sense. (Base persist directory for tasks is set by runtime properties and always overridden via `withBasePersistDirectory`; so it shouldn't be provided in the task.)

Just remove basePersistDirectory from the constructor and let it stay null here.